mwclient/client.py

```diff
@@ -1459,16 +1769,18 @@ def watchlist(self, allrev=False, start=None, end=None, namespace=None, dir='old
         return listing.List(self, 'watchlist', 'wl', max_items=max_items,
                             api_chunk_size=api_chunk_size, **kwargs)
 
-    def expandtemplates(self, text, title=None, generatexml=False):
+    def expandtemplates(
+        self, text: str, title: Optional[str] = None, generatexml: bool = False
+    ) -> Any:
```

Comment:
I think we can make this stricter - the return value should always be a string or a 2-tuple of strings. (this function definitely needs updating - the format it's using is deprecated - and probably needs more error checking - it's pretty gung-ho about assuming it gets an "OK" response!)

Reply:
Yep, I've updated this.